controls/cis_ubuntu2404.yml

```diff
@@ -2126,13 +2126,14 @@ controls:
     notes: TODO. Rule does not seem to be implemented, nor does it map to any rules in ubuntu2204 profile.
 
   - id: 5.4.2.4
-    title: Ensure root account access is controlled (Automated)
+    title: Ensure root account access is controlled
     levels:
       - l1_server
       - l1_workstation
-    status: planned
-    notes: TODO. Rule does not seem to be implemented, nor does it map to any rules in ubuntu2204 profile.
-
+    rules:
+      - ensure_root_access_controlled
+    status: manual
```

Comment:
The rule should be marked as automated because it has a working oval.
There should be a short explanation why we can't remediate in the `notes:`,
and in `warnings:` inside `rule.yml` (see for example sudoers_no_root_target, sshd_use_approved_kex_ordered_stig)

Reply:
Fixed